orchagent/pfcactionhandler.cpp

```diff
@@ -143,16 +143,32 @@ PfcWdAclHandler::PfcWdAclHandler(sai_object_id_t port, sai_object_id_t queue,
 
     // There is one handler instance per queue ID
     string queuestr = to_string(queueId);
-    m_strTable = "Table_PfcWdAclHandler_" + queuestr;
-    m_strRule = "Rule_PfcWdAclHandler_" + queuestr;
+    m_strIngressTable = "IngressTable_PfcWdAclHandler_" + queuestr;
```

Comment:
If there are 2 maps, you can still use original strings.

Reply:
AclOrch requires unique acl table name, so we need different names for ingress/egress acl table. Separating into two maps won't help. To reduce complexity a little bit, I use the same acl Rule name for both acl tables.